src/EnvRegistry.ts

```diff
@@ -12,11 +12,11 @@ import {
   EnvironmentsResponse,
   RegisterResponse,
 } from './api/EnvRegistry';
-import { isEnvKeyValid, isEnvValid, isRegisterRequestValid } from './helpers/validators';
+import { isEnvKeyValid, isRegisterRequestValid } from './helpers/validators';
 import { ConfigEntry, EntriesResponse } from './types';
 import { validationMessages } from './helpers/constants';
 
-export class EnvRegistry implements EnvRegistryInterface {
+export class EnvRegistry<T> implements EnvRegistryInterface<T> {
```

Comment:
I know that it's very common to call these vars as T, but I would prefer the name Env to explain what it represents

Reply:
Done